java/opendcs/src/main/java/org/opendcs/units/Constants.java

```diff
@@ -29,5 +29,11 @@ public class Constants {
     /* P0 is standard pressure in mb.*/
     public static final double p0 = 1013.25;
 
+    // Private constructor to prevent instantiation
+    private Constants()
+        {
```

Comment:
This applies to almost every file but the indentation here is wrong. The open curly should be under the p in private. The closing curly should be under the open curly as one would expect.